crates/ruff_linter/src/rules/isort/rules/organize_imports.rs

```diff
@@ -80,6 +82,7 @@ fn matches_ignoring_indentation(val1: &str, val2: &str) -> bool {
 #[allow(clippy::cast_sign_loss, clippy::too_many_arguments)]
 /// I001
 pub(crate) fn organize_imports(
+    path: &Path,
```

Comment:
I think I'd prefer to pass in the `TargetVersion` instead of the `path`, considering that the `path` isn't needed here otherwise.